test/unit/css.js

```diff
@@ -1323,11 +1323,11 @@ QUnit.test( "Keep the last style if the new one isn't recognized by the browser
 	assert.equal( el.css( "position" ), "absolute", "The old style is kept when setting an unrecognized value" );
 	el = jQuery( "<div></div>" ).css( "position", "absolute" ).css( "position", " " );
 
-	// Support: Edge 14
+	// Support: Edge 14-15
```

Comment:
If yes, then this already solved with SI style, since they also were concern with negative values, it's also looks confusing, this why I proposed to use ` to ` as they do